lib/git_repository/cloning.rb

```diff
@@ -58,14 +58,16 @@ def svn_has_trunk?
 
   # Sets the reference of a local branch
   def reset_branch(old_head_oid, branch, ref)
-    with_head_change old_head_oid do
-      git_exec 'branch', '-f', branch, ref
+    with_head_change(old_head_oid) do
+      git_exec('branch', '-f', branch, ref)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.